src/legacy/ui/public/styles/_mixins.scss

```diff
@@ -23,7 +23,7 @@
   }
 }
 
-@mixin kibana-resizer($size: ($euiSizeM + 2px), $direction: horizontal) {
+@mixin kibanaResizer($size: ($euiSizeM + 2px), $direction: horizontal) {
```

Comment:
Can you add a comment for what this mixin does? Have we generally been using `kbn` for naming of these general mixins?